Home page basic design

mloyanich opened the pull request:
Nothing fancy, just laying down the components on the page

Commit:
Home page basic design

pachkovska commented on src/App.css:
```diff
@@ -1,38 +1,91 @@
+}
+
+.app-tracker-add {
+  display: flex;
```
I would recommend also adding `flex-wrap: wrap` property to this one. That way logo will move to the next line on a smaller screen instead of getting cut off.

mloyanich replied:
added

pachkovska commented on src/App.css:
```diff
@@ -1,38 +1,91 @@
+  display: flex;
+}
+.add-tracker-input {
+  width: 481px;
```
I also wouldn't recommend hardcoding this value in px. I'd go either with fraction of the view width, or %, or do flex on the parent and then use flex-grow: 1.

mloyanich replied:
I put % for now, will need to redesign it once we are reusing the component

pachkovska reviewed (approved):
I would slightly correct CSS I mentioned in the comments. Overall looks good.

pachkovska commented on src/App.js:
```diff
@@ -1,11 +1,26 @@
+
+      </div>
+      <div className='app-tracker-add'>
+        <div className='add-tracker'>
+          <input type="text" className='add-tracker-input' />
+          <button className='add-tracker-button'>Track</button>
+        </div>
```
Later, I would take this out into a separate component. This will eventually need local state and handlers or some sort of form management library.

mloyanich replied:
created AddTracker component

Commit:
addressing pull request comments
Created new AddTracker component,
switched to folder per component, as we can use separate css per component
